### PR TITLE
docs: translate pricing page (part4)

### PR DESCRIPTION
**Pull request by d-bytebase**

### CODE REVIEW

File: common/plan.ts [hunk 1]
```diff
@@ -116,17 +118,17 @@ const FREE_PLAN: Plan = {
...
   trialPrice: 14,
   pricePerInstancePerMonth: 29,
-  priceDescription: "5 minimum, billed annually",
+  priceDescription: "team-price-intro",
   // Plan desc and feature
-  title: "Team",
-  description:
-    "Medium size team, with dedicated DBA or TL for engineering velocity",
+  name: "Team",
+  title: "subscription.plan.team.title",
```
Comment (ecmadao) [inline common/plan.ts:130]: The name and the title is some kind of duplicated. Maybe just keep the title?

File: pages/pricing/index.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
         <div class="relative space-y-6 lg:space-y-0 lg:grid lg:grid-cols-3">
           <div
             v-for="plan in plans"
-            :key="plan.title"
+            :key="plan.name"
```
Comment (ecmadao) [inline pages/pricing/index.vue:35]: Maybe just keep the :key="plan.title" is good enough?

**Review by ecmadao — COMMENTED** [review]
LGTM otherwise

File: common/plan.ts (hunk 1 above)
Comment (d-bytebase) [reply common/plan.ts:130]: Unfortunately, we have to keep the name for generating the icon name. (line59 in pages/pricing/index.vue)

Comment (ecmadao) [reply common/plan.ts:130]: I see.. How about using the type to build the image path? (But we need to rename these images as the plan type is the number enum)

File: pages/pricing/index.vue (hunk 2 above)
Comment (d-bytebase) [reply pages/pricing/index.vue:35]: Done.

File: common/plan.ts (hunk 1 above)
Comment (d-bytebase) [reply common/plan.ts:130]: I changed to imagePath to be more explicit about the icon path.

**Review by ecmadao — DISMISSED** [review]

File: pages/pricing/index.vue [hunk 3]
```diff
@@ -500,13 +505,19 @@ export default defineComponent({
     XIcon,
   },
   setup() {
+    const app = useContext();
```
Comment (boojack) [inline pages/pricing/index.vue:508]: It should be `const { app } = useContext();`
https://sourcegraph.com/github.com/bytebase/bytebase.com/-/blob/components/DocsSidebar.vue?L150

Comment (d-bytebase) [reply pages/pricing/index.vue]: Done.

**Review by ecmadao — APPROVED** [review]

**Pull request merged**